readme.md

```diff
@@ -68,6 +68,19 @@ chalk.green(
 	chalk.blue.underline.bold('with a blue substring') +
 	' that becomes green again!'
 );
+
+// ES2015 template literals
+chalk.red(
+	`I am a red line ${calk.blue.underline.bold('with a blue substring')} ` +
```

Comment:
Using string concatenation is not a good example of template literals as their main use is not having to do that.